src/backend/MediatypeS11n.cc

```diff
@@ -0,0 +1,89 @@
+//
+//  backend/MediatypeS11n.cc
```

Comment:
```suggestion
//  backend/MediaTypeS11n.cc
```

I find it weird that "type" is lowercased part of media type, I think this was a thing in the past implementation for Media type too though so perhaps it doesn't make sense to just change this here and leave it for consistency. Since "media type" is not one word.